src/main/java/com/checkout/workflows/four/actions/response/WorkflowActionInvocation.java

```diff
@@ -0,0 +1,25 @@
+package com.checkout.workflows.four.actions.response;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Data;
+
+import java.sql.Timestamp;
+
+@Data
+public class WorkflowActionInvocation {
+
+    @SerializedName("invocation_id")
+    private String invocationId;
+
+    private Timestamp timestamp;
+
+    private Boolean retry;
+
+    private Boolean succeeded;
+
+    @SerializedName("final")
+    private Boolean finalAttempt;
+
+    @SerializedName("result_details")
+    private String resultDetails;
```

Comment:
This should be `@SerializedName("result")` and a `Map<String, Object>`.

Reply:
It is a Map yes, but the name is result_details
https://api-reference.checkout.com/preview/crusoe/#tag/Workflows/paths/~1workflows~1events~1{eventId}~1actions~1{workflowActionId}/get!c=200&path=action_invocations/result_details&t=response